CMakeLists.txt

```diff
@@ -115,11 +115,14 @@ if(ENABLE_LTO)
 endif()
 
 if(ENABLE_OPENMP)
-    find_package( OpenMP REQUIRED )
-    include(FindOpenMP)
```

Comment:
Is this not needed? (I do not understand CMake much)

Reply:
No, this looks like a relic from the past.
'find_package' will do the same.